.sass-lint.yml

```diff
@@ -0,0 +1,182 @@
+# sass-lint config generated by make-sass-lint-config v0.1.2
```

Comment:
sass-lint has support for two different file formats `sass-lint.yml` and `scss-lint.yml` which seem to be the standard. 

If you look at Airbnb's style guide they use the scss-lint format:
https://github.com/airbnb/css

I mainly chose it because I have used it in the past and it has worked for me. Easy setup and configuration.